### PR TITLE
[965] - [Bugtask] the border should also be red for the outdated due date

### PR DESCRIPTION
**Pull request by rogeliojohnoliverio**
## Issue Link
- https://app.asana.com/0/1203011167276287/1203212353604965/f

## Definition of Done
- [x] Fix the red border color  on calendar when the  for the outdated due date

## Notes
- None

## Pre-condition
- login and go to the project board
- click any eye icon of a task
- modify the due date to outdated

## Expected Output
- Border color of the calendar should be in red

## Screenshots

https://user-images.githubusercontent.com/110364637/197149475-889a5263-60bf-4399-9cde-4d2e3ae89c9e.mp4

### CODE REVIEW

File: api/database/seeders/ProjectSeeder.php [hunk 1]
```diff
@@ -74,13 +73,13 @@ public function run()
...
-      'project_member_id' => $members->first()->id,
-      'section_id' => $sections->first()->id
-    ]);
+    // $sections = Section::factory(4)->create([
+    //   'project_id' => $project->id
+    // ]);
+    // Task::factory(4)->create([
+    //   'project_member_id' => $members->first()->id,
+    //   'section_id' => $sections->first()->id
+    // ]);
```
Comment (abduljalilpalala) [inline api/database/seeders/ProjectSeeder.php]: Remove commented lines in this file if not needed

**Review by abduljalilpalala — COMMENTED** [review]
Check comments 🗒️

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✔️

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Pull request merged**